aopy/visualization.py

```diff
@@ -1,17 +1,25 @@
 # visualization.py
 # code for general neural data plotting (raster plots, multi-channel field potential plots, psth, etc.)
 
+import seaborn as sns
 import matplotlib
 import matplotlib.pyplot as plt
 from matplotlib.animation import FuncAnimation
+
 from scipy.interpolate import griddata
 from scipy.interpolate.interpnd import _ndim_coords_from_arrays
 from scipy.spatial import cKDTree
 import numpy as np
 import os
 import copy
+
+
+from aopy import precondition
```

Comment:
this is incorrect, can't import aopy from within aopy
the correct line is 'from . import precondition'